Cloud Application Manager/Administering Your Organization/manage-assets-monitor-usage.md

```diff
@@ -9,19 +9,19 @@
 
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Manage Details](#manage-details)
-* [Manage Features](#manage-features)
-* [Manage Administrators](#manage-administrators)
-* [Manage Clouds](#manage-clouds)
-* [Manage Tags](#manage-tags)
-* [Manage Webhooks](#manage-webhooks)
-* [Manage ITSM Integrations](#manage-itsm-integrations)
-* [Manage Authentication](#manage-authentication)
-* [Manage Theme](#manage-theme)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```

Comment:
Pleae, leave the * for the bullet points (it might be another config parameter of the MD ToC plugin)